azure/container-registry/variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "resource_group_name" {
+  type = string
+}
+
+variable "name" {
+  type = string
+}
+
+variable "region" {
+  type = string
+}
+
+variable "project" {
+  type = string
+}
+
+variable "environment" {
+  type = string
+}
+
+variable "admin_enabled" {
+  type    = bool
+  default = true
+}
+
+variable "public_network_access_enabled" {
+  type    = bool
+  default = true
+}
```

Comment:
same here: good for now, but let's add an issue to disable public access, and instead use a private endpoint.